### DISCUSSION TITLE
Need `includeForks=true` for forked repository?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#20517] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I'm helping to get Renovate configured for a big repository again. 😄 

They want:

- No noise
- Only get updates manually via Dependency Dashboard
- Run Renovate on their _fork_ of the Atom editor project

The repository is: https://github.com/pulsar-edit/pulsar/ that's a fork of the Atom editor project. The official Atom project is now deprecated and won't get any updates.

I'm suggesting they use this config to start with:

```json
{
  "extends": ["config:base", ":dependencyDashboardApproval"],
  "includeForks": true
}
```

I'm worrying a bit that setting `includeForks=true` will affect the downstream forks of `pulsar-edit/edit`, for example:

1. `includeForks=true` in `renovate.json` for `pulsar-edit/edit`
1. `@username` forks `pulsar-edit/edit` into their own namespace
1. Because `@username` also uses Renovate hosted app, and allows Renovate to run in all their repositories, and because `includeForks=true` in the config: get Renovate dependency dashboard issue on their fork

I think this is the key bit of information from our docs for `includeForks`: [^docs]

> If you are using the hosted Mend Renovate then this option will be configured to `true` automatically if you "Selected" repositories individually but remain as `false` if you installed for "All" repositories.

[^docs]: https://docs.renovatebot.com/configuration-options/#includeforks

### What you tried so far.

Nothing bad happened yet, but I'm trying to prevent noise or trouble for downstream forks of the upstream project.

It appears the `pulsar-edit/edit` project is asking for GitHub to make the repository a source repository instead of a fork. Then they would not need to have `includeForks` hence no problems for downstream forks?

### Relevant debug logs

There are no logs yet, as the bot hasn't run on the repository.

**Comment by rarkins** [comment, marked as answer]
Are they using the app?

**Reply by HonkingGoose** [reply]
I'm nudging them into using the GitHub-hosted Renovate app, because that's easiest to use and explain.

**Reply by rarkins** [reply]
In this case includeForks=true is set automatically by the bot if they "Select repositories" when installing the app. They only need to set `includeForks` themselves if they choose "All repositories".

**Reply by HonkingGoose** [reply]
I'll let them select the repositories individually then. 😉 Thank you for the quick response! 😄
